packages/storybook/stories/Feedback.stories.mdx

```diff
@@ -1,25 +1,44 @@
+import { DefaultButton, Stack } from '@fluentui/react';
 import { Meta } from '@storybook/addon-docs';
 
 <Meta id="feedback" title="Feedback" />
 
 # Feedback
 
 In order to improve the UI Library for Azure Communication Services we need your feedback!
-There are various ways for you to provide feedback of your experience and help us make the UI Library better.
 
-## Contributing to UI Library
-
-If there is a feature or bug you find and you are intersted in contributing directly to our code base please refer to our [contribution guide](https://github.com/Azure/communication-ui-library/blob/main/docs/contributing-guide/1.%20getting-set-up.md).
-We encourage developers to feel welcomed into the code base and directly contribute.
-
-## GitHub Issues
-
-File feature requests, bugs or other issues against the [UI Library GitHub Repo](https://github.com/Azure/communication-ui-library/issues).
-To help us get to your issue faster use the GitHub labels provided.
-A team member will look to help out with outstanding issues and navigate them to the right owner.
-Issues on this repo should only be related to the UI Library. Issues for Azure Communication Services generally should be added to the main [repo](https://github.com/Azure/communication).
-
-## Microsoft Q&A
-
-For any questions on UI Library or Azure Communication Services, use [Microsoft Q&A](https://docs.microsoft.com/answers/topics/azure-communication-services.html).
-There use the tag `azure-communication-services` for the question to be funneled to our team.
+There are various ways for you to provide feedback of your experience and help us make the UI Library better,
+like contributing directly to our code base, report an issue or ask for new feature on our Github repo, or check
+Microsoft Q&A using the tag `azure-communication-services`.
+
+<Stack horizontal horizontalAlign="center" tokens={{ childrenGap: '4rem' }} style={{ marginTop: '5rem' }}>
+  <Stack.Item align="center">
+    <DefaultButton
+      href={
+        'https://github.com/Azure/communication-ui-library/blob/main/docs/contributing-guide/1.%20getting-set-up.md'
+      }
+      target={'_blank'}
+      iconProps={{ iconName: 'CodeEdit' }}
+      text={'Contributing'}
+      primary={true}
+    />
+  </Stack.Item>
+  <Stack.Item align="center">
+    <DefaultButton
+      href={'https://github.com/Azure/communication-ui-library/issues'}
+      target={'_blank'}
+      iconProps={{ iconName: 'Warning' }}
+      text={'GitHub report'}
```

Comment:
s/GitHub report/Report an issue ?

Actions should be imperative statements.

Reply:
You could be a writer @prprabhu-ms :D